contracts/FlowIDTableStaking.cdc

```diff
@@ -1125,11 +1142,16 @@ pub contract FlowIDTableStaking {
                 //  - New ANs may be unapproved, but must have submitted sufficient stake
                 if nodeRecord.role == UInt8(5) && !greaterThanMin && !nodeIsApproved {
                     self.removeAndRefundNodeRecord(nodeID)
+                    if participantList[nodeRecord.id] != nil {
```

Comment:
is this check needed?
why not use remove instead of setting to nil?

Reply:
Good idea. I'll make the change